src-v2/Judge/NTS.Judge.Blazor/Pages/Dashboard/Components/Actions/EliminationForms/FailedToQualifyForm.razor

```diff
@@ -1,53 +1,52 @@
 ﻿@using NTS.Domain.Core.Aggregates.Participations
 @using NTS.Domain.Objects
 @using NTS.Judge.Blazor.Enums
+@using NTS.Judge.Blazor.Pages.Dashboard.Components.Actions.EliminationForms.Components
 @using NTS.Judge.Blazor.Ports
 @using Not.Structures
 
 @inherits EliminationForm
 
-<MudSelect @bind-SelectedValues="@Codes" Label="Failed to Qualify Reason" MultiSelection="true" Delimiter="+" Placeholder="Select a reason">
-    @foreach (var failToQualifyCode in SelectListModel.FromEnum<FTQCodes>())
-    {
-        <MudSelectItem Value="@failToQualifyCode.Value">
-            <MudText Typo="Typo.body2">@failToQualifyCode.Value</MudText>
-            <MudText Typo="Typo.subtitle2">@failToQualifyCode.Description</MudText>
-        </MudSelectItem>
-    }
-</MudSelect>
+<MudPaper Elevation="1" Class="pa-2">
+    <MudSelect @bind-SelectedValues="@_codes" MultiSelection="true" Delimiter="+" Class="mt-4"
+        Label="@Localizer.Get("Failed to Qualify Reason")" Placeholder="@Localizer.Get("Select a reason")">
+    
+        @foreach (var ftqCode in SelectListModel.FromEnum<FTQCodes>())
+        {
+            <MudTooltip Text="@ftqCode.Description" Inline="false">
```

Comment:
The root of the MudTooltip gets a class
`
    .mud-tooltip-root { width: fit-content; } 
`
This causes the width of the MudSelectItem to be half of its container with the remaining space appearing blank. To prevent this something along the lines of 
`RootStyle="width:100%">` is needed.